src/composable/orderTypes/Twap.spec.ts

```diff
@@ -219,3 +221,115 @@ describe('To String', () => {
     )
   })
 })
+
+describe('Poll Validate', () => {
+  const blockNumber = 123456
+  const blockTimestamp = 1700000000
+  const mockStartTimestamp: jest.MockedFunction<(params: OwnerContext) => Promise<number>> = jest.fn()
+  const mockEndTimestamp: jest.MockedFunction<(startTimestamp: number) => number> = jest.fn()
+  const mockGetBlock: jest.MockedFunction<
+    (
+      blockHashOrBlockTag: providers.BlockTag | string | Promise<providers.BlockTag | string>
+    ) => Promise<providers.Block>
+  > = jest.fn()
+
+  const provider = {
+    getBlock: mockGetBlock,
+  } as any as providers.Provider
```

Comment:
Just to get rid of this annoying warning

```suggestion
  } as unknown as providers.Provider
```